command.go

```diff
@@ -204,7 +206,11 @@ func (c *Command) runWithResult(cmd *exec.Cmd) (*Result, error) {
 
 	cmd.Stdout = stdout
 	cmd.Stderr = stderr
+	if runtime.GOOS == "windows" {
```

Comment:
rather than checking `runtime.GOOS`, can you move this logic into the `command_windows.go` file I created in this PR? Should no longer need `runtime.GOOS` checks in that case. I don't know if the `CreationFlags` field exists on all systems, so that would be safer, as it won't even do AST checking for non-windows.